pkg/reconciler/taskrun/resources/pod.go

```diff
@@ -50,6 +50,8 @@ const (
 	taskRunLabelKey     = pipeline.GroupName + pipeline.TaskRunLabelKey
 	ManagedByLabelKey   = "app.kubernetes.io/managed-by"
 	ManagedByLabelValue = "tekton-pipelines"
+
+	scriptsDir = "/builder/scripts"
```

Comment:
We really need to discussed reserved folder (and document those) aka #1030
I would love to have `/tekton/scripts` instead of `/builder` but meh :stuck_out_tongue_closed_eyes:

Reply:
+1

I'm not sure the best place to document that. I'd be in favor of renaming the reserved directory too, in a later change.

Reply:
#1304 has been open for a bit now...I'll add it to next week's WG

Reply:
that should be in the beta list too